californium-core/src/main/java/org/eclipse/californium/core/CoapServer.java

```diff
@@ -147,7 +147,7 @@ public CoapServer(final NetworkConfig config, final int... ports) {
 		}
 		
 		// resources
-		this.root = createRoot();
+		this.root = new RootResource();
```

Comment:
Maybe it is more about providing means for subclasses to provide a different *banner* for `GET /` requests? That could be easily achieved ...